docs/source/installation.rst

```diff
@@ -207,6 +207,22 @@ Some users have experienced issues when installing NumPy, particularly version 1
     Another option under Windows is to follow the instructions for `Unix-like OS`_, using the Windows Subsystem for Linux (WSL).
     For installing WSL, follow the instructions `here <https://docs.microsoft.com/en-us/windows/wsl/install-win10#step-2---check-requirements-for-running-wsl-2>`_.
 
+Some quick tips on virtual environments
```

Comment:
not a good header, very vague. Headers should be concrete and short. Can you find sth clearer?

Reply:
Replaced with `Setting up a virtual environment`